rep-0151.rst

```diff
@@ -175,7 +175,7 @@ Shebangs and reliance on the Python command
 '''''''''''''''''''''''''''''''''''''''''''
 Python scripts on UNIX systems typically have shebang lines written as:
 
-.. code-block: bash
+  .. code-block:: python
```

Comment:
I don't think this is correct.  The code we are looking at is bash code, not python.  I think this should be:

```suggestion
.. code-block:: bash
```

Reply:
Ah that makes sense.

I think I checked the change I made renders using Ubuntu local rendering tool (I think `typora` but I forgot), but `bash` makes better sense.